File: rest-boot/src/main/java/com/example/model/Car.java
package com.example.model;

public class Car {

	int vin;
	String model;
	String make;
	double price;

	public Car() {
	}

	public Car(String model, String make, double price) {
		this.model = model;
		this.make = make;
		this.price = price;
	}

	public Car(int vin, String model, String make, double price) {
		this.vin = vin;
		this.model = model;
		this.make = make;
		this.price = price;
	}

	public int getVin() {
		return vin;
	}

	public void setVin(int vin) {
		this.vin = vin;
	}

	public String getModel() {
		return model;
	}

	public void setModel(String model) {
		this.model = model;
	}

	public String getMake() {
		return make;
	}

	public void setMake(String make) {
		this.make = make;
	}

	public double getPrice() {
		return price;
	}

	public void setPrice(double price) {
		this.price = price;
	}

	@Override
	public String toString() {
		return "Car [vin=" + vin + ", model=" + model + ", make=" + make + ", price=" + price + "]";
	}

}

File: Day3-Before/Before Lunch/car.ts
export class Car {

	vin:number = 1000;
	model:string = "X5";
	make:string = "BMW";
	price:number = 7800000.00;
	color:string = "Black";
	logo:string = "path";

	constructor(vin,model,make,price,color,logo){
		this.vin = vin;
		this.model = model;
		this.make = make;
		this.price = price;
		this.color = color;
		this.logo = logo;
	}
}

